External/FEXCore/Source/Interface/IR/PassManager.cpp

```diff
@@ -2,28 +2,34 @@
 #include "Interface/IR/Passes/RegisterAllocationPass.h"
 #include "Interface/IR/PassManager.h"
 
+#include <FEXCore/Config/Config.h>
+
 namespace FEXCore::IR {
 
 void PassManager::AddDefaultPasses(bool InlineConstants, bool StaticRegisterAllocation) {
-  InsertPass(CreateContextLoadStoreElimination());
-  InsertPass(CreateDeadFlagStoreElimination());
-  InsertPass(CreateDeadGPRStoreElimination());
-  InsertPass(CreateDeadFPRStoreElimination());
-  InsertPass(CreatePassDeadCodeElimination());
-  InsertPass(CreateConstProp(InlineConstants));
+  FEXCore::Config::Value<bool> DisablePasses{FEXCore::Config::CONFIG_DEBUG_DISABLE_OPTIMIZATION_PASSES, false};
+
+  if (!DisablePasses()) {
+    InsertPass(CreateContextLoadStoreElimination());
+    InsertPass(CreateDeadFlagStoreElimination());
+    InsertPass(CreateDeadGPRStoreElimination());
+    InsertPass(CreateDeadFPRStoreElimination());
+    InsertPass(CreatePassDeadCodeElimination());
+    InsertPass(CreateConstProp(InlineConstants));
 
-  ////// InsertPass(CreateDeadFlagCalculationEliminination());
+    ////// InsertPass(CreateDeadFlagCalculationEliminination());
 
-  InsertPass(CreateSyscallOptimization());
-  InsertPass(CreatePassDeadCodeElimination());
+    InsertPass(CreateSyscallOptimization());
+    InsertPass(CreatePassDeadCodeElimination());
 
-  // only do SRA if enabled and JIT
-  if (InlineConstants && StaticRegisterAllocation)
-    InsertPass(CreateStaticRegisterAllocationPass());
+    // only do SRA if enabled and JIT
+    if (InlineConstants && StaticRegisterAllocation)
```

Comment:
SRA can't be turned off right now. It's on my todo list.

Reply:
Fixed